rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -1,557 +1,585 @@
-/*
```

Comment:
Seems that you changed the encoding for this file and it's impossible to review. Please fix.